### PR TITLE
Introduce TS-like model

### PR DESCRIPTION
**Pull request by qwrtln**
An attempt at data model

### CODE REVIEW

File: src/main/vue/src/models/CategoryModel.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export class Category {
+
+    public static fromArray(rawCategories: any[]): Category[] {
+        const categories = [] as Category[];
+        rawCategories.forEach(r => {
```
Comment (szewa-91) [inline src/main/vue/src/models/CategoryModel.ts]: You can use map() here

File: src/main/vue/src/components/RelaxContent.vue [hunk 2]
```diff
@@ -14,14 +15,16 @@
 <script lang="ts">
   import { Component, Prop, Vue } from 'vue-property-decorator';
   import SoundTile from '@/components/SoundTile.vue';
+  
+  import { Category } from '@/models/CategoryModel.ts';
```
Comment (szewa-91) [inline src/main/vue/src/components/RelaxContent.vue]: 'Model' in file name seems redundant

File: src/main/vue/src/models/CategoryModel.ts (hunk 1 above)
Comment (qwrtln) [reply src/main/vue/src/models/CategoryModel.ts]: :+1:

File: src/main/vue/src/components/RelaxContent.vue (hunk 2 above)
Comment (qwrtln) [reply src/main/vue/src/components/RelaxContent.vue]: :+1:

**Pull request merged**